mappings/net/minecraft/command/CommandBase.mapping

```diff
@@ -1,27 +1,162 @@
 	METHOD a getMinPermissionLevel ()I
+	METHOD a normalizeBlockPosSelector (DLjava/lang/String;IIZ)Laq$a;
+		ARG 0 originalPos
+		ARG 1 selector
+		ARG 2 min
+		ARG 3 max
+		ARG 4 middle
+	METHOD a normalizeBlockPosSelector (DLjava/lang/String;Z)Laq$a;
+		ARG 0 originalPos
+		ARG 1 selector
+		ARG 2 middle
 	METHOD a toBlockState (Lank;Ljava/lang/String;)Lave;
+		ARG 0 block
+		ARG 1 meta
 	METHOD a setCommandManager (Lap;)V
+		ARG 0 commandManager
 	METHOD a compareTo (Las;)I
+		ARG 0 command
+	METHOD a getPlayerFromSender (Lau;)Lnn;
+		ARG 0 sender
 	METHOD a dispatch (Lau;Las;ILjava/lang/String;[Ljava/lang/Object;)V
 	METHOD a dispatch (Lau;Las;Ljava/lang/String;[Ljava/lang/Object;)V
+	METHOD a giveItem (Lau;Ljava/lang/String;)Lahd;
+		ARG 0 sender
+		ARG 1 itemName
+	METHOD a toBlockPos (Lau;[Ljava/lang/String;IZ)Ldx;
+		ARG 0 sender
+		ARG 1 params
+		ARG 2 startIndex
+		ARG 3 middle
+	METHOD a getStateWithProperty (Lave;Lavu;Ljava/lang/Comparable;)Lave;
+		ARG 0 blockState
+		ARG 1 propertyKey
+		ARG 2 propertyValue
+	METHOD a toBlockPropertyvalue (Lavu;Ljava/lang/String;)Ljava/lang/Comparable;
+		ARG 0 blockProperty
+	METHOD a getCommandJsonException (Lcom/google/gson/JsonParseException;)Ldq;
+		ARG 0 parseException
 	METHOD a toInteger (Ljava/lang/String;)I
+		ARG 0 toConvert
 	METHOD a toDoubleWithMin (Ljava/lang/String;D)D
+		ARG 0 toConvert
+		ARG 1 min
 	METHOD a toDoubleWithBounds (Ljava/lang/String;DD)D
+		ARG 0 toConvert
+		ARG 1 min
+		ARG 2 max
 	METHOD a toIntegerWithMin (Ljava/lang/String;I)I
+		ARG 0 toConvert
+		ARG 1 min
 	METHOD a toIntegerWithBounds (Ljava/lang/String;II)I
+		ARG 0 toConvert
+		ARG 1 min
+		ARG 2 max
 	METHOD a toLongWithBounds (Ljava/lang/String;JJ)J
+		ARG 0 toConvert
+		ARG 1 min
+		ARG 2 max
 	METHOD a matchesBeginning (Ljava/lang/String;Ljava/lang/String;)Z
+		ARG 0 string
+		ARG 1 toFind
+	METHOD a joinPrettyStringCollection (Ljava/util/Collection;)Ljava/lang/String;
+		ARG 0 collection
 	METHOD a joinPretty (Ljava/util/List;)Lgk;
+		ARG 0 list
 	METHOD a canExecute (Lnet/minecraft/server/MinecraftServer;Lau;)Z
+		ARG 0 server
+		ARG 1 sender
+	METHOD a getPlayerFromUuid (Lnet/minecraft/server/MinecraftServer;Lau;Ljava/lang/String;)Lnn;
+		ARG 0 server
+		ARG 1 sender
+		ARG 2 uuid
+	METHOD a getEntityFromUuid (Lnet/minecraft/server/MinecraftServer;Lau;Ljava/lang/String;Ljava/lang/Class;)Lub;
+		ARG 0 server
+		ARG 1 sender
+		ARG 2 uuid
+		ARG 3 entityClass
 	METHOD a getTabCompletionList (Lnet/minecraft/server/MinecraftServer;Lau;[Ljava/lang/String;Ldx;)Ljava/util/List;
+		ARG 0 server
+		ARG 1 sender
+		ARG 2 commandList
+		ARG 3 pos
+	METHOD a getEntityTagCompound (Lub;)Lfc;
+		ARG 0 entity
+	METHOD a joinPrettyString ([Ljava/lang/Object;)Ljava/lang/String;
+		ARG 0 list
+	METHOD a joinStringArray ([Ljava/lang/String;I)Ljava/lang/String;
+		ARG 0 list
+		ARG 1 startIndex
+	METHOD a getFullBlockPosComponent ([Ljava/lang/String;ILdx;)Ljava/util/List;
```

Comment:
Do all these methods need to end in `BlockPosComponent`? Would just `Pos` suffice?